packages/prisma/schema.prisma

```diff
@@ -1605,18 +1607,24 @@ enum EventTypeAutoTranslatedField {
 }
 
 model EventTypeTranslation {
-  id             String                       @id @default(cuid())
+  uid            String                       @id @default(cuid())
```

Comment:
Need to keep `id` here for now until the second release where we drop it.

Reply:
ah, stupid mistake, addressed